benchmarks/AutomaticDifferentiation/Project.toml

```
@@ -27,7 +27,7 @@ Zygote = "e88e6eb3-aa80-5325-afca-941959d7151f"
 BenchmarkTools = "1"
 DataFrames = "1"
 DataFramesMeta = "0.15"
-DifferentiationInterface = "0.9"
+DifferentiationInterface = "0.6"
```

```suggestion
DifferentiationInterface = "0.7"
```
unless something is blocking it?

---

Something is.